documentation/projects/proposals/ingestion_server_removal/20240328-implementation_plan_ingestion_server_removal.md

```diff
@@ -609,3 +617,52 @@ monitor the first production data refreshes closely.
 
 [^2]:
     https://towardsdatascience.com/using-apache-airflow-dockeroperator-with-docker-compose-57d0217c8219
+
+[^3]:
+    See #4456 for further context on this. The filtering is a _necessary_ step
+    of the data refresh we need to carry forward even after removing the other
+    cleanup steps. There are a number of ways we can accomplish this, but by far
```

Comment:
This looks too long to be a footnote. Wouldn't it be better under the "Add remaining steps to the Data Refresh DAGs" section in the filtering step?

Reply:
If we want to make the footnote smaller I think it would be better move the specific steps for the task into `Create the new data refresh DAG factory and move initial steps into Airflow` (since this is one of the initial steps, even though it's going to end up being implemented after the others) and the suggested future optimizations into `Alternatives` or something like that.

Not remotely a blocker for me but might make it easier to find when reading back through this document later.

Reply:
Thanks you two for mentioning it - part of why I wanted to add it as a footnote was because this context has been added after-the-fact, and having it literally at the end of the document in the form of a footnote visually and conceptually matches that intent! I can move some of the information around as suggested, then merge this in 😄

Reply:
Done in d84ddde!